easybuild/easyconfigs/g/gtest/gtest-1.8.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = "Tarball"
+
+name = 'gtest'
```

Comment:
@hajgato Why does this also include `gtest`, not a dep?

Reply:
@boegel First I thought that `gtest` is a dep, but it is only needed if you run the tests. By default, tests are not run, so I removed the `gtest` dep. Once I made the EasyConfig, I did not want to waste it. If its not a big problem, then I would not make a separate PR for it.

Reply:
OK, it's fine to leave it there, I thought it got included by accident.